sunsynk/sensor.py

```diff
@@ -241,17 +241,90 @@ def update_value(self) -> None:
             _LOGGER.error("Could not decode temperature: %s", err)
 
 
+class Minutes(int):
+    """Deals with inverter time format conversion complexities."""
+
+    @classmethod
+    def from_str_value(cls, str_value: str) -> Minutes:
+        """Create from a string in hh:mm format."""
+        return cls(int(str_value[:-3]) * 60 + int(str_value[-2:]))
+
+    @classmethod
+    def from_reg_value(cls, reg_value: int) -> Minutes:
```

Comment:
The reg_value can also be rework so that you can assign to it with `@reg_value.setter`